scripts/format-changelogs.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+FILES=("$@")
+
+if [[ ${#FILES[@]} -eq 0 ]]
+then
+  mapfile -t FILES < <(git ls-files '*CHANGELOG.md')
+fi
+
+cardano-ledger-release-tool changelogs -i "${FILES[@]}"
```

Comment:
We should add functionality for finding all changelogs into the tool itself. This would make `-i` argument optional (eg. if you want to run it on a specific file), but by default it would just recursively find all `changelog.md` files disregarding the case.
Conduit has a nice function that can make this task very easy: [`sourceDirectoryDeep`](https://www.stackage.org/haddock/nightly-2025-09-25/conduit-extra-1.3.8/Data-Conduit-Filesystem.html#v:sourceDirectoryDeep)

This approach would make presence of this script redundant.

Reply:
FTR: https://github.com/input-output-hk/cardano-ledger-release-tool/issues/1